src/test/java/seedu/mtracker/console/AddForexParserTest.java

```diff
@@ -74,6 +70,29 @@
             + SEPARATOR_SPECIFIER + FUTURE_DATE
             + SEPARATOR_SPECIFIER + "fooRemarks";
 
+    public static final String USER_INPUT_TRY_INVALID_EXPIRY = SEPARATOR_SPECIFIER + "TTTXXX"
+            + SEPARATOR_SPECIFIER + "0.81"
+            + SEPARATOR_SPECIFIER + "foobar"
+            + SEPARATOR_SPECIFIER + "negative"
+            + SEPARATOR_SPECIFIER + "0.79"
+            + SEPARATOR_SPECIFIER + "0.70"
+            + SEPARATOR_SPECIFIER.repeat(2) + "testDate"
+            + SEPARATOR_SPECIFIER + "31/12/2021"
+            + SEPARATOR_SPECIFIER + "2021-01-01"
```

Comment:
Use past date here

Reply:
Sorry I wanted to put "2021.01.01" actually. PAST_DATE is used in USER_INPUT_TRY_PAST_EXPIRY below.